verification/curator-service/ui/cypress/integration/components/LinelistTableTest.spec.ts

```diff
@@ -9,6 +9,15 @@ describe('Linelist table', function () {
         cy.clearSeededLocations();
     });
 
+    it('Display case properly', function () {
+        cy.addCase('France', 'some notes', 'www.example.com');
+        cy.visit('/cases');
+        cy.contains('PCR test');
```

Comment:
It's not the best to test for something that wasn't explicitly set in the setup. WDYT about adding this to the addCase parameters? We could change the params to opts so people only have to set what they are explicitly testing for.

Reply:
good call, done.